request_a_govuk_domain/request/utils.py

```diff
@@ -76,7 +76,7 @@ def route_number(session_data: dict) -> dict[str, int]:
             route["primary"] = 1
             if session_data.get("domain_confirmation") == "no":
                 route["secondary"] = 12
-        elif registrant_type in ["central_government", "ndpb"]:
+        elif registrant_type in ["central_government", "alb"]:
```

Comment:
When the method is_central_government was written, it was being used in more than one place. But I don't see that is the case anymore. So either we can use the method call here, or remove the method is_central_government